### DISCUSSION TITLE
whether k3s  support kubeflow?

### DISCUSSION THREAD

**Discussion by iambyd** [k3s-io/k3s#3062] [Q&A]
I want to use kubeflow,but k8s install is so difficult，and i use k3s do this thing？

**Comment by brandond** [comment]
It appears that kubeflow can be installed with kustomize, and k3s supports kustomize the same as upstream Kubernetes... so yeah, it should work.

**Comment by iambyd** [comment]
ok, i just the first Guinea pig, thanks!

**Comment by theofpa** [comment]
it just takes 4 lines to do so:
```
curl -sfL https://get.k3s.io | sh -
git clone git@github.com:kubeflow/manifests.git && cd manifests
while ! kustomize build example | sudo k3s kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
sudo k3s kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
```
visit http://localhost:8080
username: `user@example.com`
password: `12341234`
